### PR TITLE
Add multi-tenant capabilities to sample POS's

### PR DESCRIPTION
**Pull request by mattgresham**
- Add multi-tenant capability to RamenPOS (SP-674)
- Add multi-tenant capability to TablePOS (SP-674)
- Add multi-tenant capability to MotelPOS (SP-674)

### CODE REVIEW

File: motelpos/Podfile [hunk 1]
```diff
@@ -1,12 +1,15 @@
...
+platform :ios, '9.0'
 
 target 'MotelPos' do
     # Comment the next line if you're not using Swift and don't want to use dynamic frameworks
     use_frameworks!
     
     # Pods for MotelPos
-    pod 'SPIClient-iOS', '~> 2.6.4'
+    # pod 'SPIClient-iOS', '~> 2.6.6'
+    pod 'SPIClient-iOS', :git => 'https://github.com/mx51/spi-client-ios.git', :branch => 'feature/v2.6.6'
```
Comment (mattgresham) [inline motelpos/Podfile]: This reference will be replaced with the previous line before merging

File: ramenpos/Podfile [hunk 2]
```diff
@@ -1,12 +1,13 @@
...
+platform :ios, '9.0'
 
 target 'RamenPos' do
     # Comment the next line if you're not using Swift and don't want to use dynamic frameworks
     use_frameworks!
     
     # Pods for RamenPos
-    pod 'SPIClient-iOS', '~> 2.6.4'
+    # pod 'SPIClient-iOS', '~> 2.6.6'
+    pod 'SPIClient-iOS', :git => 'https://github.com/mx51/spi-client-ios.git', :branch => 'feature/v2.6.6'
```
Comment (mattgresham) [inline ramenpos/Podfile]: This reference will be replaced with the previous line before merging

File: tablepos/Podfile [hunk 3]
```diff
@@ -1,12 +1,13 @@
...
+platform :ios, '9.0'
 
 target 'TablePos' do
     # Comment the next line if you're not using Swift and don't want to use dynamic frameworks
     use_frameworks!
 
     # Pods for TablePos
-    pod 'SPIClient-iOS', '~> 2.6.4'
+    # pod 'SPIClient-iOS', '~> 2.6.6'
+    pod 'SPIClient-iOS', :git => 'https://github.com/mx51/spi-client-ios.git', :branch => 'feature/v2.6.6'
```
Comment (mattgresham) [inline tablepos/Podfile]: This reference will be replaced with the previous line before merging

File: tablepos/TablePos/View Controllers/Connection/ConnectionViewController.swift [hunk 4]
```diff
@@ -177,5 +204,35 @@ class ConnectionViewController: UITableViewController, NotificationListener {
             self.txtOutput.text = msg + "\n================\n" + self.txtOutput.text
         }
     }
+
+    func getTenantCode(tenantName: String) -> String? {
+        return TableApp.current.settings.tenantList.first(where:{$0["name"] == tenantName})!["code"]
+    }
+}
+
+extension ConnectionViewController: UIPickerViewDelegate, UIPickerViewDataSource {
```
Comment (mattgresham) [inline tablepos/TablePos/View Controllers/Connection/ConnectionViewController.swift]: This doesn't feel right. Any suggestions on this are most welcome.

**Review by gouline — APPROVED** [review]
🚀

**Pull request merged**